chapters/0100-introduction/text.md

```diff
@@ -3,4 +3,37 @@
 
 ## Introduction
 
+## What to expect
 
+This book presents and advocates for a principled and rigorous approach to
+software development. Instead of focusing on finding *a* solution to a problem,
+we'll be searching for the *best* solution. This kind of endeavor necessarily
+requires a certain amount of discipline and effort, but your labors will be
+rewarded handsomely.
+
+Some of this work borrows from academia and, consequently, we'll be introducing
+and using some academic terms. At times we'll even use mathematical notation
+when we need the precision it offers.
+
+This being said, our goal is not to produce a purely theoretical exercise coming
+from yet another ivory tower. If this work can be only one thing, it must be
+preeminently practical. Our goal in creating this material is that you will
+henceforth code differently . . . better.
+
+Euclid famously once said "there is no royal road to geometry." Likewise,
+though we've done our best to make this work as accessible as possible, to get
+the most out of this book you're going to have to engage deeply with the
+material. The included exercises are necessary to hone your skills.
+
+We'll be using terms in this book that you probably recognize like "semantics",
+"contracts", "safety" and "algorithm". While we try to distill the general sense
+of these words into something precise, our definitions may not perfectly match
+your intuition. We mention this so that you don't mistake our usage of these
+words with anything other than the precise definitions we provide.
+
+It is also important to forewarn you that we're going to take some strong
+stances against some of today's commonly accepted practices. You, yourself, are
+likely doing some of these things on a regular basis. We urge you to keep an
+open mind as you reflect upon our suggested alternatives. You can expect that it
+may take some time reviewing code in practice before you reach the same
+conclusions we have.
```

Comment:
I'm a bit concerned that the confidence of this statement is approaching arrogance. 

> "If you don't see the brilliance of our conclusions, it's probably because you haven't reviewed as much code as we have."

I suggest adding a statement as a counterbalance to this, which emphasizes the co-creative nature of programming truth:

```suggestion
conclusions we have. Of course, you might never agree with our conclusions. We encourage you to rigorously develop your criticisms of our work, and invite you into the uniquely cooperative pursuit of programming truth. 
```